contrib/kind.sh

```diff
@@ -391,7 +391,7 @@ if [ "${GITHUB_ACTIONS:-false}" == "true" ]; then
       -e 's/^.*kubernetes cluster\.local/& net/' \
       -e '/^.*upstream$/d' \
       -e '/^.*fallthrough.*$/d' \
-      -e '/^.*forward . \/etc\/resolv.conf$/d' \
+      -e '/^.*forward . 8.8.8.8' \
```

Comment:
this isnt a valid regex, you need to substitute correctly here

Reply:
done